src/overlays/actors/ovl_En_Invadepoh_Demo/z_en_invadepoh_demo.h

```diff
@@ -2,15 +2,54 @@
 #define Z_EN_INVADEPOH_DEMO_H
 
 #include "global.h"
+#include "assets/objects/gameplay_keep/gameplay_keep.h"
+#include "assets/objects/object_cow/object_cow.h"
+#include "assets/objects/object_ma1/object_ma1.h"
+#include "assets/objects/object_uch/object_uch.h"
+
+typedef enum {
+    /* 0 */ EN_INVADEPOH_DEMO_TYPE_ALIEN,
+    /* 1 */ EN_INVADEPOH_DEMO_TYPE_ROMANI,
+    /* 2 */ EN_INVADEPOH_DEMO_TYPE_COW,
+    /* 3 */ EN_INVADEPOH_DEMO_TYPE_UFO,
+    /* 4 */ EN_INVADEPOH_DEMO_TYPE_COW_TAIL,
+    /* 5 */ EN_INVADEPOH_DEMO_TYPE_MAX
+} EnInvadepohDemoType;
+
+#define EN_INVADEPOH_DEMO_GET_TYPE(thisx) ((thisx)->params & 0xF)
+#define EN_INVADEPOH_DEMO_GET_CUE_ID_OFFSET(thisx) (((thisx)->params >> 11) & 0x7)
+#define EN_INVADEPOH_DEMO_GET_PATH_INDEX(thisx) (((thisx)->params >> 4) & 0x7F)
+#define EN_INVADEPOH_DEMO_GETZ_SPEED(thisx) ((thisx)->shape.rot.z)
```

Comment:
Oh interesting that it pulls from shape. I think in most cases, even if params are stored in `(thisx)->shape.rot.` or `(thisx)->world.rot.`, we still use `_GET_` in the params, but I can see how `en_bom.h` uses the `_GETX_` and `_GETZ_`. Not sure if it's important enough to distinguish, thoughts?

Reply:
I don't really think the distinction is really that big of a deal. Additionally, literally everything *but* EnBom just ignores this. I'll remove it (from this and EnBom too).